### PR TITLE
[Backports] Shader Graph FogBugz Fixes

### PR DESCRIPTION
**Pull request by alindmanUnity**
### Purpose of this PR
Why is this PR needed, what hard problem is it solving/fixing?

Backports the following bug fixes:
FB#1131211 - Voronoi Node Serialization Issue PR #3138
FB#1131755 - Drag-dropping normal map texture into a shader graph causes errors PR #3137
FB#1132342 - Multiply Bug. Zeroing components in code generation PR #3134
FB#1131646 - IsFrontFace Breaks Subgraphs PR #3133

---
### Release Notes
Please add any useful notes about the feature/fix that might be helpful for others.

---
### Testing status
**Katana Tests**: First off we need to make sure the Katana SRP tests are green?
All Tests Green
https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=sg%2Fbug-fix-backports&unity_branch=2019.1%2Fstaging&automation-tools_branch=master

**Manual Tests**: What did you do?
- [x] Opened test project + Run graphic tests locally
- [x] C# and shader warnings (supress shader cache to see them)
- Other: 
Opened project and hand-tested each fix multiple times to confirm errors are gone.

**Automated Tests**: What did you setup?

Any test projects to go with this to help reviewers?

---
### Overall Product Risks
**Technical Risk**: None, Low, Medium, High? Low

**Halo Effect**: None, Low, Medium, High? Low

---
### Comments to reviewers
Notes for the reviewers you have assigned.

### CODE REVIEW

File: com.unity.shadergraph/CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [5.7.0] - 2019-XX-XX
+### Fixed
+- Fixed an error when opening saved Shader Graphs containing one or more Voronoi nodes.
```
Comment (sharlenet) [inline com.unity.shadergraph/CHANGELOG.md]: If the user is the one opening the Shader Graphs then:
Fixed an error that previously occurred when you opened saved Shader Graphs containing one or more Voronoi nodes.
Else:
Fixed an error that previously occurred when opening saved Shader Graphs containing one or more Voronoi nodes.

File: com.unity.shadergraph/CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- You can now drag normal map type textures on to a Shader Graph to create Sample Texture 2D nodes with the correct type set.
```
Comment (sharlenet) [inline com.unity.shadergraph/CHANGELOG.md]: Minor change - made '"onto" one word:

You can now drag normal map type textures onto a Shader Graph to create Sample Texture 2D nodes with the correct type set.

File: com.unity.shadergraph/CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- Fixed the Multiply node so default input values are applied correctly.
+- Fixed an issue where IsFaceSign would not compile within SubGraph Nodes.
```
Comment (sharlenet) [inline com.unity.shadergraph/CHANGELOG.md]: Added back-ticks, and split Sub Graph into two words to match the rest of docs:

Fixed an issue where `IsFaceSign` would not compile within Sub Graph Nodes.

File: com.unity.shadergraph/CHANGELOG.md [hunk 4]
```diff
@@ -5,6 +5,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+- The Sub Graph Outputs node can no longer be duplicated.
```
Comment (sharlenet) [inline com.unity.shadergraph/CHANGELOG.md]: You can no longer duplicate the Sub Graph Outputs node.

**Comment by zadi0648** [comment]
Waiting on 3144 before approving

**Pull request merged**